eng/codeql.ps1

```diff
@@ -0,0 +1,195 @@
+function Install-Gdn {
+    param(
+        [string]$Path
+    )
+
+    Start-Process nuget -Verbose -ArgumentList "install", "Microsoft.Guardian.Cli", "-Source https://securitytools.pkgs.visualstudio.com/_packaging/Guardian/nuget/v3/index.json", "-OutputDirectory $Path", "-NonInteractive", "-NoCache" -NoNewWindow -Wait
+    $gdnCliPath = Get-ChildItem -Filter guardian.cmd -Recurse -Path $Path
+    return $gdnCliPath.FullName
+}
+
+function Initialize-Gdn {
+    param(
+        [Parameter(Mandatory)]
+        [string]$GuardianCliLocation,
+        [Parameter(Mandatory)]
+        [string]$WorkingDirectory,
+        [ValidateSet("Trace", "Verbose", "Standard", "Warning", "Error")]
+        [string]$LoggerLevel = "Standard"
+    )
+
+    if (!(Test-Path $GuardianCliLocation)) {
+        throw "GuardianCliLocation not found"
+    }
+
+    & $GuardianCliLocation init --working-directory $WorkingDirectory --logger-level $LoggerLevel
+}
+
+function New-GdnSemmelConfig {
```

Comment:
Nit:  Should this be "New-GdnSemmleConfig"?

Reply:
Yep. Spelling 🔤.